### DISCUSSION TITLE
[BUG] Error 1. Retrying in five minutes.

### DISCUSSION THREAD

**Discussion by Alexey018n** [Chia-Network/chia-blockchain#7337] [Support]
All time, time by time a have this error. I have 1Tb Disk for farming. I try farm with another PC buy have error 1. Plotting not go up after 1%. What i can do with this chia coins?
***
I can repair plotting.
The problem was resolved by changing the carrier letter.
Initially, 1 disk was split into 2 (X, Y), where X was temporary, Y was permanent. Added a second Z drive. And it allowed the system to run normally.
But the synchronization has been going on for 20 hours already = (
How many times need for synchronization k=32, 101,4 GiB?

**Comment by TXAlexX** [comment]
I have have also been getting this error (Error 1. Retrying in five minutes.) since upgrading to Version 1.2.0, although, I get past the 1% point.  I have plenty of disk space, but have been unable to add a single plot using the new version.
I will try on another computer when I get the chance.

**Comment by TXAlexX** [comment]
To be more specific: My current plot is frozen at 49% complete.  It has been throwing the error for 2.5 hours.  My Temp drive has 530 GB free.  
This is the 3rd plot to fail in this way.  I did not record the details of the previous 2 failures, but, plotting was around 35% complete in those cases.

**Comment by loppefaaret** [comment]
@Alexey018n could you provide the actual error message from the log please ?

**Comment by TXAlexX** [comment]
> @Alexey018n could you provide the actual error message from the log please ?

I'm starting to think this might be a drive beginning to fail, and the new version of Chia is just a coincidence. Anyway, I ran another plot. This one failed at 62%. Here is the last page or so of the log file:

Bucket 126 uniform sort. Ram: 15.490GiB, u_sort min: 1.125GiB, qs min: 0.562GiB.
	Bucket 127 uniform sort. Ram: 15.490GiB, u_sort min: 1.125GiB, qs min: 0.562GiB.
	Total matches: 4293824325
Forward propagation table time: 2196.051 seconds. CPU (700.300%) Fri Jul  9 23:21:35 2021
Time for phase 1 = 16235.490 seconds. CPU (587.000%) Fri Jul  9 23:21:35 2021

Starting phase 2/4: Backpropagation into tmp files... Fri Jul  9 23:21:35 2021
Backpropagating on table 7
scanned table 7
scanned time =  123.320 seconds. CPU (47.080%) Fri Jul  9 23:23:39 2021
sorting table 7
Backpropagating on table 6
scanned table 6
scanned time =  220.372 seconds. CPU (99.070%) Fri Jul  9 23:32:48 2021
sorting table 6
sort time =  907.488 seconds. CPU (84.620%) Fri Jul  9 23:47:56 2021
Backpropagating on table 5
scanned table 5
scanned time =  209.662 seconds. CPU (99.850%) Fri Jul  9 23:51:27 2021
sorting table 5
sort time =  816.101 seconds. CPU (89.840%) Sat Jul 10 00:05:03 2021
Backpropagating on table 4
scanned table 4
scanned time =  210.880 seconds. CPU (99.790%) Sat Jul 10 00:08:36 2021
sorting table 4
sort time =  836.082 seconds. CPU (86.570%) Sat Jul 10 00:22:32 2021
Backpropagating on table 3
scanned table 3
scanned time =  213.617 seconds. CPU (99.870%) Sat Jul 10 00:26:08 2021
sorting table 3
sort time =  804.481 seconds. CPU (89.210%) Sat Jul 10 00:39:32 2021
Backpropagating on table 2
Only read 0 of 1048576 bytes at offset 1370475762 from "D:\\Chia\\plot-k32-2021-07-09-18-50-a05e162daeb281802d708a65ac8d47930e461931d838d29c49218fe2c2d49e23.plot.table2.tmp" with length 22282002144. Error 1. Retrying in five minutes.
Only read 0 of 1048576 bytes at offset 1370475762 from "D:\\Chia\\plot-k32-2021-07-09-18-50-a05e162daeb281802d708a65ac8d47930e461931d838d29c49218fe2c2d49e23.plot.table2.tmp" with length 22282002144. Error 1. Retrying in five minutes.
Only read 0 of 1048576 bytes at offset 1370475762 from "D:\\Chia\\plot-k32-2021-07-09-18-50-a05e162daeb281802d708a65ac8d47930e461931d838d29c49218fe2c2d49e23.plot.table2.tmp" with length 22282002144. Error 1. Retrying in five minutes.
Only read 0 of 1048576 bytes at offset 1370475762 from "D:\\Chia\\plot-k32-2021-07-09-18-50-a05e162daeb281802d708a65ac8d47930e461931d838d29c49218fe2c2d49e23.plot.table2.tmp" with length 22282002144. Error 1. Retrying in five minutes.
Only read 0 of 1048576 bytes at offset 1370475762 from "D:\\Chia\\plot-k32-2021-07-09-18-50-a05e162daeb281802d708a65ac8d47930e461931d838d29c49218fe2c2d49e23.plot.table2.tmp" with length 22282002144. Error 1. Retrying in five minutes.
Only read 0 of 1048576 bytes at offset 1370475762 from "D:\\Chia\\plot-k32-2021-07-09-18-50-a05e162daeb281802d708a65ac8d47930e461931d838d29c49218fe2c2d49e23.plot.table2.tmp" with length 22282002144. Error 1. Retrying in five minutes.

**Comment by Alexey018n** [comment]
I can repair plotting.
The problem was resolved by changing the carrier letter.
Initially, 1 disk was split into 2 (X, Y), where X was temporary, Y was permanent. Added a second Z drive. And it allowed the system to run normally.
But the synchronization has been going on for 20 hours already = (
How many times need for synchronization k=32, 101,4 GiB?

**Reply by loppefaaret** [reply]
it really depends on your setup, and how you configure the plotters use of drives - normally, it would take 6-12 hours on an SSD, and 8-24 hours on HDD

**Comment by BrandtH22** [comment]
Since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
